### PR TITLE
Add `Resurrection Cup 2024` news post

### PR DESCRIPTION
**Pull request by itsmehoaq**
[web preview](https://osu2.roanh.dev/home/news/2024-05-12-resurrection-cup-2024-registration-now-open), [#osu-news thread](https://discord.com/channels/188630481301012481/1233349822359273504)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] website is up and running (registrations etc.)
- [x] upload video asset

### CODE REVIEW

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+layout: post
+title: "Resurrection Cup 2024: Registrations Now Open!"
+date: 2024-05-12 12:00:00 +0000
+---
+
+Resurrecting all osu! players! Resurrection Cup is back with our 2024 iteration - now with lots of improvement behind the scenes, focusing on competitiveness and enjoyment for everyone!
+
+![](https://hoaq.s-ul.eu/6IRgB87D)
+
+**[Resurrection Cup](https://osu.ppy.sh/community/forums/topics/1919672?n=1)**, or ResCup for short, is an annual open-rank TeamVs tournament for every player, with a max team size of 8 players, bringing a breath of fresh air to the top-tier competitive tournament scene. Every year, we release many customs - both songs and maps, with the goal of creating unique challenges for the players competing in the tournament. You may want to watch the following video, showcasing our notable custom songs from the 2023’s iteration, and expect even higher quality of production to this year’s iteration:
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:11]:
```suggestion
**[Resurrection Cup](https://osu.ppy.sh/community/forums/topics/1919672?n=1)**, or ResCup for short, is an annual open-rank Team Vs tournament featuring teams of up to 8 players each, bringing a breath of fresh air to the top-tier competitive tournament scene. Every year, we release many customs — both songs and maps — with the goal of creating unique challenges for the players competing in the tournament. You may want to watch the following video, showcasing our notable custom songs from the 2023’s iteration, and expect an even higher quality of production to this year’s iteration:
```

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://hoaq.s-ul.eu/vkUyKgyg" type="video/mp4" preload="none">
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:15]: @Walavouchey is using s-ul links in news post acceptable? i don't think i've ever seen any other news posts prior that links directly to s-ul

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## How do I register?
+
+To register, head over to our **[Registration page](https://www.rescup.xyz/register)** to connect your osu! and Discord account, as well as setting up your team! Before registering, please make sure that:
+- You must not have violated the **[osu! community rules](https://osu.ppy.sh/wiki/en/Rules)** within the past 12 months.
+- You joined the **[official Discord server](https://discord.gg/UNzyfgGfeu)** for ResCup24.
+    
+On top of that, you are required to sign up with a team of at least 5 members. If you don’t have enough team members, the `#looking-for-team` channel in our **[Discord server](https://discord.gg/UNzyfgGfeu)** should be where you headed first before signing up!
+
+We also encourage spectators to join **[ResCup Discord server](https://discord.gg/UNzyfgGfeu)** in order to keep up with all important announcements and livestream notification. For more information, check out our **[forum post](https://osu.ppy.sh/community/forums/topics/1919672?n=1)** and the **[tournament website](https://www.rescup.xyz/)**. We hope to see you join, either as players or spectators!
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:44]:
```suggestion
We also encourage spectators to join the **[ResCup Discord server](https://discord.gg/UNzyfgGfeu)** in order to keep up with all important announcements and livestream notifications. For more information, check out our **[forum post](https://osu.ppy.sh/community/forums/topics/1919672?n=1)** and the **[tournament website](https://www.rescup.xyz/)**. We hope to see you around, either as a player or spectator!
```

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+| Quarterfinals | July 5 - July 7 |
+| Semifinals | July 12 - July 14 |
+| Finals | July 19 - July 21 |
+| Grand Finals | July 26 - July 28 |
+
+## How do I register?
+
+To register, head over to our **[Registration page](https://www.rescup.xyz/register)** to connect your osu! and Discord account, as well as setting up your team! Before registering, please make sure that:
+- You must not have violated the **[osu! community rules](https://osu.ppy.sh/wiki/en/Rules)** within the past 12 months.
+- You joined the **[official Discord server](https://discord.gg/UNzyfgGfeu)** for ResCup24.
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:40]:
```suggestion
- You have joined the **[ResCup24 official Discord server](https://discord.gg/UNzyfgGfeu)**.
```

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://hoaq.s-ul.eu/vkUyKgyg" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## What's new?
+Compared to 2023’s iteration, the match procedure and rescheduling stayed the same. However, we do have a lot more surprises this time around. Many new mappers and artists joined our production team to bring you over 20 memorable custom songs and over 80 beatmaps designed and fine-tuned by experienced mappoolers and tournament players as playtesters. You can head over to our **[website](https://www.rescup.xyz/)** to find out more details about the overall mappool structure, as well as other related rules.
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:20]:
```suggestion
The match procedure and the rescheduling method for the tournament remains the same as the 2023 iteration. However, we do have a lot of surprises up our sleeve this time around. Many new mappers and artists have joined our production team, bringing you over 20 memorable custom songs and over 80 beatmaps designed and fine-tuned by experienced mappoolers and playtesters. You can head over to our **[website](https://www.rescup.xyz/)** to find out more details about the overall mappool structure as well as other relevant rules.
```

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Resurrection Cup 2024: Registrations Now Open!"
+date: 2024-05-12 12:00:00 +0000
+---
+
+Resurrecting all osu! players! Resurrection Cup is back with our 2024 iteration - now with lots of improvement behind the scenes, focusing on competitiveness and enjoyment for everyone!
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:7]:
```suggestion
Resurrecting all osu! players! Resurrection Cup is back for our 2024 iteration — now with tons of improvements behind the scenes, focusing on competitiveness and enjoyment for everyone!
```

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+| Finals | July 19 - July 21 |
+| Grand Finals | July 26 - July 28 |
+
+## How do I register?
+
+To register, head over to our **[Registration page](https://www.rescup.xyz/register)** to connect your osu! and Discord account, as well as setting up your team! Before registering, please make sure that:
+- You must not have violated the **[osu! community rules](https://osu.ppy.sh/wiki/en/Rules)** within the past 12 months.
+- You joined the **[official Discord server](https://discord.gg/UNzyfgGfeu)** for ResCup24.
+    
+On top of that, you are required to sign up with a team of at least 5 members. If you don’t have enough team members, the `#looking-for-team` channel in our **[Discord server](https://discord.gg/UNzyfgGfeu)** should be where you headed first before signing up!
```
Comment (Nivalyx) [inline news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:42]:
```suggestion
On top of that, you are required to sign up with a team of at least 5 members. If you don’t have enough team members, the `#looking-for-team` channel in our **[Discord server](https://discord.gg/UNzyfgGfeu)** should be where you are heading first before signing up for the tournament!
```

File: news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md (hunk 2 above)
Comment (Walavouchey) [reply news/2024/2024-05-12-resurrection-cup-2024-registration-now-open.md:15]: no it'll get uploaded to assets.ppy.sh

**Review by Walavouchey — DISMISSED** [review]

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**